app/src/main/res/values-es/strings.xml

```diff
@@ -1,8 +1,8 @@
 <resources>
-<string name="app_name">Sentido del rastro</string>
+<string name="app_name">Sentido de orientación</string>
```

Comment:
I'll approve this change because it was already translated to Spanish, but I'm going to keep the app name untranslated in the future as per https://github.com/kylecorry31/Trail-Sense/issues/96

Reply:
(I never added it to the README's contribution section)